elements/lisk-cryptography/src/encrypt.ts

```diff
@@ -100,13 +101,13 @@ export const encryptAES128GCMWithPassword = async (
 	const SALT_BUFFER_SIZE = 8;
 	const salt = crypto.randomBytes(SALT_BUFFER_SIZE);
 	const iv = crypto.randomBytes(IV_BUFFER_SIZE);
-	const iterations =
-		kdf === KDF.ARGON2 ? ARGON2_ITERATIONS : options?.kdfparams?.iterations ?? PBKDF2_ITERATIONS;
```

Comment:
I think the issue was when setting the iteration, it was not taking the iteration. otherwise it should use the default for each kdf, so intended condition was something like this?
```
const iterations = options?.kdfparams?.iterations ?? kdf === KDF.ARGON2 ? ARGON2_ITERATIONS : PBKDF2_ITERATIONS;
```